File: core/src/main/java/ru/hits/core/domain/dto/account/MasterAccountResponseWrapper.java
package ru.hits.core.domain.dto.account;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class MasterAccountResponseWrapper {
    private boolean success;
    private AccountDTO data;
    private String errorMessage;
}

File: loan service/HITS bank/Controllers/Dto/Message/AccountDto/MasterAccountResponseWrapper.cs
using System.Text.Json.Serialization;

namespace HITS_bank.Controllers.Dto.Message.AccountDto;
#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member

public class MasterAccountResponseWrapper
{
    [JsonPropertyName("success")]
    public bool Success { get; set; }
    
    [JsonPropertyName("data")]
    public AccountDto Data { get; set; }
    
    [JsonPropertyName("errorMessage")]
    public string ErrorMessage { get; set; }
}